queries.sql

```diff
@@ -140,3 +140,72 @@ LEFT JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
+
+
+--Who was the last animal seen by William Tatcher?
+SELECT a.name
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+INNER JOIN vets vt ON v.vet_id = vt.id
+WHERE vt.name = 'Vet William Tatcher'
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+--How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT v.animal_id) AS animal_count
+FROM visits v
+INNER JOIN vets vt ON v.vet_id = vt.id
+WHERE vt.name = 'Vet Stephanie Mendez';
+
+--List all vets and their specialties, including vets with no specialties.
+SELECT v.name, COALESCE(s.name, 'No specialty') AS specialty
+FROM vets v
+LEFT JOIN specializations sp ON v.id = sp.vet_id
+LEFT JOIN species s ON sp.species_id = s.id;
+
+--List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT a.name
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+INNER JOIN vets vt ON v.vet_id = vt.id
+WHERE vt.name = 'Vet Stephanie Mendez'
+  AND v.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
+
+--What animal has the most visits to vets?
+SELECT a.name, COUNT(v.animal_id) AS visit_count
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+GROUP BY a.name
+ORDER BY visit_count DESC
+LIMIT 1;
+
+--Who was Maisy Smith's first visit?
+SELECT a.name
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+INNER JOIN vets vt ON v.vet_id = vt.id
+WHERE vt.name = 'Vet Maisy Smith'
+ORDER BY v.visit_date ASC
+LIMIT 1;
+
+--Details for most recent visit: animal information, vet information, and date of visit.
+SELECT a.name AS animal_name, ve.name AS vet_name, v.visit_date
+FROM visits AS v
+INNER JOIN animals AS a ON v.animal_id = a.id
+INNER JOIN vets AS ve ON v.vet_id = ve.id
+INNER JOIN species AS s ON a.species_id = s.id
+WHERE s.name IN ('Pokemon', 'Digimon');
+
+--How many visits were with a vet that did not specialize in that animal's species?
+SELECT COUNT(*) AS visit_count
+FROM visits v
+INNER JOIN animals a ON v.animal_id = a.id
+INNER JOIN vets vt ON v.vet_id = vt.id
+LEFT JOIN specializations sp ON vt.id = sp.vet_id AND a.species_id = sp.species_id
+WHERE sp.vet_id IS NULL;
+
+--What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+SELECT v.name AS vet_name, COALESCE(s.name, 'No specialty') AS specialty
+FROM vets AS v
+LEFT JOIN specializations AS sp ON v.id = sp.vet_id
+LEFT JOIN species AS s ON sp.species_id = s.id;
```

Comment:
- [ ] Kindly ensure that you add screenshots of their queries results to the pull request description. It is impossible to check if their results are correct. This will help us to ensure that you run the queries with success.